po/de.po

```diff
@@ -0,0 +1,455 @@
+# Martin Pitt <mpitt@redhat.com>, 2019. #zanata
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE_VERSION\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-10-30 08:05+0000\n"
+"PO-Revision-Date: 2019-10-29 08:13+0000\n"
+"Last-Translator: Martin Pitt <mpitt@redhat.com>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"X-Generator: Zanata 4.6.2\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+
+#: src/ImageDeleteModal.jsx:16
+msgid "Are you sure you want to delete this image?"
+msgstr "Sind Sie sicher, dass Sie dieses Image löschen wollen?"
+
+#: src/ContainerCommitModal.jsx:171 src/ImageDetails.jsx:28
+msgid "Author"
+msgstr "Autor"
+
+#: src/app.jsx:401
+msgid "Automatically start podman on boot"
+msgstr "podman automatisch beim Hochfahren starten"
```

Comment:
Maybe if it would start with capital letter would be better. Is this correct grammar-wise?

Reply:
Right, this may be a bit inconsistent. I guess I can't quite decide yet whether "podman" is a kind of trade mark/proper noun, or whether it's legit to capitalize it. I updated it in Zanata now.